clicommand/pipeline_upload.go

```diff
@@ -56,13 +56,14 @@ type PipelineUploadConfig struct {
 	NoInterpolation bool   `cli:"no-interpolation"`
 
 	// Global flags
-	Debug   bool   `cli:"debug"`
-	NoColor bool   `cli:"no-color"`
-	Profile string `cli:"profile"`
+	Debug   bool         `cli:"debug"`
+	NoColor bool         `cli:"no-color"`
+	Experiments []string `cli:"experiment" normalize:"list"`
+	Profile string       `cli:"profile"`
 
 	// API config
 	DebugHTTP        bool   `cli:"debug-http"`
-	AgentAccessToken string `cli:"agent-access-token"`
+	AgentAccessToken string `cli:"agent-access-token" validate:"required"`
```

Comment:
I see this brings it consistent with all the others:

![image](https://user-images.githubusercontent.com/15759/84465485-78467780-acba-11ea-828f-0510f615b0d8.png)